packages/react-components/react-radio/src/stories/Radio/RadioDefault.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { tokens, useId, Radio } from '@fluentui/react-components';
+import type { RadioProps } from '@fluentui/react-components';
+
+export const Default = (props: Partial<RadioProps>) => {
+  const radioName = useId('radio');
+  const labelId = useId('label');
+  return (
+    <div style={{ display: 'grid', gridRowGap: tokens.spacingVerticalS }}>
```

Comment:
nit: probably better to write styles as a `makeStyles()` call as I think we'd consider that a "best practice"

Reply:
All of the radio stories are doing this. I'll fix them all in a follow up PR, tracked by issue https://github.com/microsoft/fluentui/issues/25259